src/map.ts

```diff
@@ -30,11 +30,13 @@ export class MapObject extends MapLocation implements MapRect {
     public readonly id: number;
     public width = 1;
     public height = 1;
+    public readonly properties: {[key: string]: string};
```

Comment:
Should prop-editor edited state participate in undo/redo? If so, we'll need create operation objects that capture property deltas (or full states) and add those to the log. Could be done as follow up work.

Reply:
hmm yeah good question. my initial instinct is that undo/redo would only apply to like, visible map operations, BUT i'm willing to be convinced if other people fell differently